rotkehlchen/tests/api/test_caching.py

```diff
@@ -0,0 +1,181 @@
+from unittest.mock import MagicMock, patch
+import pytest
+import requests
+from rotkehlchen.api.server import APIServer
+from rotkehlchen.chain.ethereum.modules.curve.curve_cache import (
+    CurvePoolData,
+    read_curve_pools_and_gauges,
+    save_curve_data_to_cache,
+)
+from rotkehlchen.chain.ethereum.utils import should_update_protocol_cache
+from rotkehlchen.db.ens import DBEns
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.tests.utils.api import api_url_for, assert_proper_response
+from rotkehlchen.tests.utils.factories import make_evm_address
+from rotkehlchen.types import CacheInvalidationType, GeneralCacheType
+from rotkehlchen.utils.misc import ts_now
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_icons_and_avatars_cache_deletion(rotkehlchen_api_server):
+    """Checks that clearing the cache for avatars and icons work as expected."""
+    icons_dir = rotkehlchen_api_server.rest_api.rotkehlchen.icon_manager.icons_dir
```

Comment:
as discussed we are not invalidating anything, just forcing a requery of the pools. I would restore those

Reply:
THis test to where it was before

Reply:
yeahy keeping it here is fine.